lib/test.rb

```diff
@@ -0,0 +1,13 @@
+[1,2,3,4].each_with_index do |elem,index|
+puts "this is the #{index +1} iteration"
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Surrounding space missing for operator '+'.<br>Do not write to stdout. Use Rails' logger if you want to log.